samples-aspnetcore-3x/okta-hosted-login/okta-aspnetcore-mvc-example/appsettings.json

```diff
@@ -9,6 +9,7 @@
   "Okta": {
     "OktaDomain": "https://{yourOktaDomain}",
     "ClientId": "{ClientId}",
-    "ClientSecret": "{ClientSecret}"
+    "ClientSecret": "{ClientSecret}",
+    "AuthorizationServerId": "default" // Provide your Authorization Server name, or use the `default` one.
```

Comment:
@laura-rodriguez It's better to remove the comment. JSONs don't support them and we already had an issue related to comments in configs.